cs3api4lab/config/config_manager.py

```diff
@@ -77,6 +77,9 @@ class Config(LoggingConfigurable):
     oauth_token = Unicode(
         config=True, allow_none=True, help="""OAuth token"""
     )
+    dev_env = Bool(
```

Comment:
I suggest adding default value in config manager's internal json until this values mess is cleaned up

Reply:
also maybe add some description